jonprl-mode.el

```diff
@@ -85,11 +85,11 @@ manually."
   :group 'jonprl
   :options '(jonprl-update-operators))
 
-(defcustom jonprl-pre-check-buffer-hook '(save-buffer)
+(defcustom jonprl-pre-check-buffer-hook '(save-buffer jonprl-initialize-configuration)
   "A hook to run prior to checking the buffer."
   :type 'hook
   :group 'jonprl
-  :options '(save-buffer))
+  :options '(save-buffer jonprl-initialize-configuration))
```

Comment:
Would it ever make sense to not run `jonprl-initialize-configuration`? If not, then I don't think it should be here, but just be called directly where the hook is invoked.